docs/source/upgrade/example/run.sh

```diff
@@ -28,5 +28,6 @@ fi
 TESTS_DIR=$(dirname $(rlocation "$TEST_WORKSPACE/docs/source/upgrade/example/run.sh"))
 damlc=$(rlocation "$TEST_WORKSPACE/$1")
 
-(cd $TESTS_DIR/coinV1; $damlc build)
-(cd $TESTS_DIR/coinV2; $damlc build)
+(cd $TESTS_DIR/coin-1.0.0; $damlc build)
+(cd $TESTS_DIR/coin-2.0.0; $damlc build)
+(cd $TESTS_DIR/coin-upgrade; $damlc build)
```

Comment:
Now we have an example of a project which depends on *two* other projects...I'm excited that the new `daml_build_test` rule is going to get more of a workout. :)

Reply:
Worked out nicely, great work :+1: